test/integration/transaction.spec.ts

```diff
@@ -37,13 +37,13 @@ describe('liquidjs-lib (transactions with psbt)', () => {
       // non-segwit inputs now require passing the whole previous tx as Buffer
       nonWitnessUtxo: Buffer.from(
         '0200000000010caf381d44f094661f2da71a11946251a27d656d6c141577e27c483a6' +
-        'd428f01010000006a47304402205ac99f5988d699d6d9f72004098c2e52c8f342838e' +
-        '9009dde33d204108cc930d022077238cd40a4e4234f1e70ceab8fd6b51c5325954387' +
-        '2e5d9f4bad544918b82ce012102b5214a4f0d6962fe547f0b9cbb241f9df1b61c3c40' +
-        '1dbfb04cdd59efd552bea1ffffffff020125b251070e29ca19043cf33ccd7324e2dda' +
-        'b03ecc4ae0b5e77c4fc0e5cf6c95a010000000005f5df70001976a914659bedb5d3d3' +
-        'c7ab12d7f85323c3a1b6c060efbe88ac0125b251070e29ca19043cf33ccd7324e2dda' +
-        'b03ecc4ae0b5e77c4fc0e5cf6c95a010000000000000190000000000000',
+          'd428f01010000006a47304402205ac99f5988d699d6d9f72004098c2e52c8f342838e' +
```

Comment:
I don't understand all this kind of changes.... all files should be properly linted and prettiefied already.. all these changes are really annoying :/

Reply:
I only use the command `npm run format && npm run format:ci` that generates this changes. That's mandatory, the CI pipeline fails if format is not apply.